recipe/meta.yaml

```diff
@@ -25,13 +29,13 @@ source:
   #   folder: numpy/core/src/npysort/x86-simd-sort
 
 build:
-  number: 0
+  number: 1
   skip: true  # [py<39]
   entry_points:
     - f2py = numpy.f2py.f2py2e:main             # [win]
     - numpy-config = numpy._configtool:main     # [win]
   run_exports:
-    - {{ pin_subpackage("numpy") }}
+    - numpy >={{ default_abi_level }}
```

Comment:
Does this need `<3`? Or does the current invocation somehow know this already?

Reply:
Yeah, we can add this too. I had been considering if we want to set it to `<2.3`, but not sure yet

Reply:
I think `<3` is better; `<2.3` is only a good idea for packages which use a lot of NumPy's API surface and are very actively maintained.

Reply:
Done in commit: https://github.com/conda-forge/numpy-feedstock/commit/33f1ad49c5d2fe53659affc7d117f380a9d27b72